### PR TITLE
Lathe parser updates related to github issue #60

### PR DESCRIPTION
**Pull request by wfpokorny**
In summary, current on curve point checking for positive x is correct only for the cubic spline, wrong for the bezier control points and both its ends, and wrong for one or both ends of the quadratic and linear splines, respectively. The new checking forces all on curve, specified points, to be positive in x. Know that while not too likely, somebody, somewhere, no doubt specified point(s) which worked with the previous code which will now need adjustment at the first or last point.

### CODE REVIEW

File: source/parser/parser.cpp [hunk 1]
```diff
@@ -2911,7 +2911,7 @@ ObjectPtr Parser::Parse_Julia_Fractal ()
 
 ObjectPtr Parser::Parse_Lathe()
 {
-    int i;
+    int i, AlreadyWarned;
```
Comment (c-lipka) [inline source/parser/parser.cpp]: Can you make this a bool, and adjust the test and modification of this variable accordingly? I don't think we need to count how many times we have issued a warning.

**Comment by wfpokorny** [comment]
Hi Christoph, I've made the update. Wondered about updating an active pull request, but git / github made it easy. 

I saw too your requests for a documentation update and for a separate prism->bezier_spline->normal issue to be opened. I am working on a larger test case now for the new prism issue. 

Where do I make / suggest these documentation updates? Is this something that can be done through the wiki page itself? Or still the newsgroup & somebody else does the actual update? Thanks.

**Comment by c-lipka** [comment]
Once you have signed in to the Wiki, you should be able to make changes to the documentation (though some pages have been locked; if that is the case for the portions you intend to provide changes for, please add the suggested new text to the discussion page).

**Comment by c-lipka** [comment]
Whoops, wrong button.

**Pull request merged**